### ISSUE TITLE
pgdg-centos93-9.3-1 not found in yum.postgresql.org

### ISSUE THREAD

**Issue by tiago-cruz-movile** [sous-chefs/postgresql#361]
```
tiagocruz@stark:~/chef-repo/cookbooks/postgresql/attributes$ git diff
diff --git a/cookbooks/postgresql/attributes/yum_pgdg_packages.rb b/cookbooks/postgresql/attributes/yum_pgdg_packages.rb
index fda17ed..d5b7b8c 100644
--- a/cookbooks/postgresql/attributes/yum_pgdg_packages.rb
+++ b/cookbooks/postgresql/attributes/yum_pgdg_packages.rb
@@ -206,7 +206,7 @@ default['postgresql']['pgdg']['repo_rpm_url'] = {
         },
         "x86_64" => {
           "url" => "http://yum.postgresql.org/9.3/redhat/rhel-6-x86_64/",
-          "package" => "pgdg-centos93-9.3-1.noarch.rpm"
+          "package" => "pgdg-centos93-9.3-3.noarch.rpm"
         }
       },
       "5" => {
```

**Comment by sfiggins** [comment]
We noticed a problem starting yesterday where we can no longer get the noarch.rpm from pgdg.  This happens for any version of postgresql we try. We receive 404 errors: 

       remote_file[/tmp/kitchen/cache/pgdg-centos94-9.4-2.noarch.rpm] (postgresql::yum_pgdg_postgresql line 34) had an error: Net::HTTPServerException: 404 "Not Found"

We thought this was maybe some problem with an http -> https redirect, but we still have the problem after updating the uri. The error above occurred processing this resource:

       remote_file("/tmp/kitchen/cache/pgdg-centos94-9.4-2.noarch.rpm") do
         provider Chef::Provider::RemoteFile
         action "create"
         retries 0
         retry_delay 2
         guard_interpreter :default
         path "/tmp/kitchen/cache/pgdg-centos94-9.4-2.noarch.rpm"
         backup 5
         atomic_update true
         source ["https://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-2.noarch.rpm"]
         use_etag true
         use_last_modified true
         cookbook_name "postgresql"
         recipe_name "yum_pgdg_postgresql"
         mode "0644"
       end

**Comment by tiago-cruz-movile** [comment]
Maybe add some like 

 `not_if 'rpm -qa pgdg-centos93'`

in remote_file resource https://github.com/hw-cookbooks/postgresql/blob/develop/recipes/yum_pgdg_postgresql.rb#L26-L29 ?

**Comment by sfiggins** [comment]
We have discovered for us that this is something that was fixed by using a later version. The servers having the problem were running an older version of chef 11.  Our problem is possibly a duplicate of CHEF-5010.  11.16.4 or 11.18.12 worked.

**Comment by legal90** [comment]
Hi, all!  There is a workaround for this issue. Just override the following attribute on any upper level:

```
node.default['postgresql']['pgdg']['repo_rpm_url']['9.3']['centos']['6']['x86_64'] = 'http://yum.postgresql.org/9.3/redhat/rhel-6-x86_64/pgdg-centos93-9.3-3.noarch.rpm'
```

**Comment by tas50** [comment]
@tiago-cruz-movile What version of this cookbook are you using?

**Comment by tiago-cruz-movile** [comment]
@tas50,

```
tiagocruz@stark:~/chef-repo/cookbooks$ grep version postgresql/metadata.rb 
version           "4.0.6"
```

Thanks

**Comment by damacus** [comment]
Hi @tiago-cruz-movile looks like you're using an old version of the cookbook. Can you update and let us know the result please?

**Comment by damacus** [comment]
Closed as 9.3 is no longer available and EOL.
